### DISCUSSION TITLE
Can we implement Drag and Drop for columns in FluentDataGrid?

### DISCUSSION THREAD

**Discussion by thieng101** [microsoft/fluentui-blazor#3067] [Q&A]
Hello I have the question as the title.

**Comment by dvoituron** [comment]
I suppose that you want to re-order the columns?

No that is not included and planned.

**Reply by thieng101** [reply]
@dvoituron thank you. Additional question, are we able to apply different text colors for the headers and the rows ? (still using FluentDataGrid and PropertyColumn)

**Reply by dvoituron** [reply]
Yes using `TemplateColumn` and `PropertyColumn`.

**Reply by thieng101** [reply]
@dvoituron I'm not very familiar with the framework. I'd very much appreciate it if you could clarify how we can use `TemplateColumn` and `PropertyColumn`.

**Reply by dvoituron** [reply]
You have samples on this page https://fluentui-blazor.net/DataGrid

And code in our GitHub repository 😀

We recorded some videos also on our website.

**Reply by thieng101** [reply]
@dvoituron thank you so much!
